File: cielo24_Java/cielo24/src/cielo24/json/Job.java
package cielo24.json;

import cielo24.utils.Guid;
import cielo24.utils.MicroDate;

import static cielo24.Enums.*;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.Hashtable;

public class Job extends JsonBase {
	
    @SerializedName("JobId")
	public Guid jobId;
	@SerializedName("JobName")
	public String jobName;
    @SerializedName("MediaLengthSeconds")
    public Float mediaLengthSeconds;
    @SerializedName("ExternalID")
    public String externalID;
    @SerializedName("Priority")
    public Priority priority;
    @SerializedName("Fidelity")
    public Fidelity fidelity;
    @SerializedName("TurnaroundTimeHours")
    public Integer turnaroundTimeHours;
	@SerializedName("JobStatus")
	public JobStatus jobStatus;
	@SerializedName("SourceLanguage")
	public String sourceLanguage;
	@SerializedName("TargetLanguage")
	public String targetLanguage;
	@SerializedName("CreationDate")
	public MicroDate creationDate;
    @SerializedName("StartDate")
    public MicroDate startDate;
	@SerializedName("DueDate")
	public MicroDate dueDate;
    @SerializedName("CompletedDate")
    public MicroDate completedDate;
    @SerializedName("ReturnTargets")
    public Hashtable<String, ArrayList<Hashtable<String, String>>> returnTargets;
    @SerializedName("Options")
    public Hashtable<String, Hashtable<String, String>> options;
}

File: cielo24_.NET/Cielo24/JSON/Job/Job.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace Cielo24.JSON.Job
{
    public class Job : JsonBase
    {
        [JsonProperty("JobId")]
        public Guid JobId { get; set; }
        [JsonProperty("JobName")]
        public string JobName { get; set; }
        [JsonProperty("MediaLengthSeconds")]
        public int? MediaLengthSeconds { get; set; }
        [JsonProperty("ExternalID")]
        public String ExternalID { get; set; }
        [JsonProperty("Priority")]
        public Priority? Priority { get; set; }
        [JsonProperty("Fidelity")]
        public Fidelity? Fidelity { get; set; }
        [JsonProperty("TurnaroundTimeHours")]
        public int? TurnaroundTimeHours { get; set; }
        [JsonProperty("JobStatus")]
        public JobStatus? JobStatus { get; set; }
        [JsonProperty("SourceLanguage")]
        public string SourceLanguage { get; set; }
        [JsonProperty("TargetLanguage")]
        public string TargetLanguage { get; set; }
        [JsonProperty("CreationDate")]
        public DateTime? CreationDate { get; set; }
        [JsonProperty("StartDate")]
        public DateTime? StartDate { get; set; }
        [JsonProperty("DueDate")]
        public DateTime? DueDate { get; set; }
        [JsonProperty("CompletedDate")]
        public DateTime? CompletedDate { get; set; }
        [JsonProperty("ReturnTargets")]
        public Dictionary<String, String> ReturnTargets { get; set; }
        [JsonProperty("Options")]
        public Dictionary<String, String> Options { get; set; }

        // BACKWARD COMPATABILITY //
        [JsonProperty("JobLanguage")]
        [Obsolete("This property is obsolete; use SourceLanguage property instead.")]
        public string JobLanguage { get; set; }
        [JsonProperty("CreationTime")]
        [Obsolete("This property is obsolete; use CreationDate property instead.")]
        public DateTime? CreationTime { get; set; }
        [JsonProperty("StartTime")]
        [Obsolete("This property is obsolete; use StartDate property instead.")]
        public DateTime? StartTime { get; set; }
        [JsonProperty("CompletedTime")]
        [Obsolete("This property is obsolete; use CompletedDate property instead.")]
        public DateTime? CompletedTime { get; set; }
    }
}